packages/material-ui/src/Hidden/HiddenCss.test.js

```diff
@@ -95,7 +87,7 @@ describe('<HiddenCss />', () => {
       const theme = createMuiTheme({ breakpoints: { keys: ['xxl'] } });
       const { container } = render(
         <MuiThemeProvider theme={theme}>
-          <HiddenCss xxlUp className="testid" classes={{ xxlUp: 'xxlUp' }}>
+          <HiddenCss xxlUp className="testid xxlUp">
```

Comment:
```suggestion
          <HiddenCss xxlUp className="testid" classes={{ xxlUp: 'xxlUp' }}>
```

Reply:
Please don't change the tests, unless necessary. The logic in the test should still work.

Reply:
I was assuming that the `classes` prop would be completely ignored as we're defining the classes ourselves, but I guess they're factored into `composeClasses()` -- that makes a lot more sense, actually!